app/common/directives/mainsheet-container.directive.js

```diff
@@ -52,15 +53,26 @@ function MainsheetContainer($rootScope, $compile, MainsheetService) {
             mainsheetContent.html(template);
             $compile(mainsheetContent)(templateScope);
 
+            $timeout(function () {
```

Comment:
Lets wait for Rominas review on my change here: https://github.com/ushahidi/platform-client/pull/1637#discussion_r564068092, and if its ok lets do the same approach to the mainsheet :)